pkg/scheduler/apis/config/v1beta3/types.go

```diff
@@ -56,6 +56,14 @@ type LoadAwareSchedulingArgs struct {
 	// EstimatedScalingFactors indicates the factor when estimating resource usage.
 	// The default value of CPU is 85%, and the default value of Memory is 70%.
 	EstimatedScalingFactors map[corev1.ResourceName]int64 `json:"estimatedScalingFactors,omitempty"`
+	// ForceEstimationSecondsAfterPodScheduled indicates the force estimation duration
+	// after pod condition PodScheduled transition to True in seconds.
+	ForceEstimationSecondsAfterPodScheduled *int64 `json:"forceEstimationSecondsAfterPodScheduled,omitempty"`
+	// ForceEstimationSecondsAfterInitialized indicates the force estimation duration
+	// after pod condition Initialized transition to True in seconds.
+	ForceEstimationSecondsAfterInitialized *int64 `json:"forceEstimationSecondsAfterInitialized,omitempty"`
```

Comment:
What is the difference between the `forceEstimationSecondsAfterPodScheduled` and the `forceEstimationSecondsAfterInitialized`? I suppose we can merge into a single one, which means estimating the pod usage based on its request, regardless of whether it has a pod metric or not.